doc/en/cache.rst

```diff
@@ -73,7 +73,7 @@ If you then run it with ``--lf``::
 
     $ py.test --lf
     ======= test session starts ========
-    platform linux -- Python 3.4.2, pytest-2.8.1.dev1, py-1.4.30, pluggy-0.3.1
+    platform linux -- Python 3.4.3, pytest-2.8.1, py-1.4.30, pluggy-0.3.1
```

Comment:
No need to update this, just thought I would mention: next time we generate the examples I think (and Ronny agreess) that we should use Python 3.5 as a rule of thumb.